src/Symfony/Component/Finder/Tests/FinderTest.php

```diff
@@ -852,4 +853,15 @@ public function testMultipleLocationsWithSubDirectories()
         $this->assertIterator($expected, $finder);
         $this->assertIteratorInForeach($expected, $finder);
     }
+
+    public function testShellCommand()
+    {
+        $cmd = Command::create()->add('--force');
+        $this->assertSame('--force', $cmd->join());
```

Comment:
Maybe it's a better idea to split this into more than one test then.

By the way, it could be interesting to use index `1` instead of `2` in the second call of `addAtIndex()` to ensure that you can also insert in the middle.